charts/newrelic-logging/templates/fluent-bit-namespace-to-api-key-mapping.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluent-bit-namespace-to-api-key-mapping
+  # TODO: do we need namespace and labels? I stole them from configmap.yml
+  namespace: {{ .Release.Namespace }}
+  labels: {{ include "newrelic-logging.labels" . | indent 4 }}
+data:
+  # TODO: this needs to be a Vault path instead of hard-coding keys
```

Comment:
I'm pretty sure that we have a way to get secrets from Vault into k8s secrets so that they can be used. I'm assuming that's possible, I'm skipping it here just so I can get done with the prototype sooner.